po/Makefile.am

```diff
@@ -27,7 +27,7 @@ src/ws/%.po: po/%.po
 	$(COPY_RULE)
 
 # Build a javascript file from a po
-po.%.js: %.po
+po.%.js: %.po $(NPM)
```

Comment:
Okay.  Now I see why you call this $(NPM).... because of the way you use it everywhere.

I wonder if you could avoid this noise with `BUILT_SOURCES`.

Reply:
@allisonkarlitskaya : I now [read the BUILT_SOURCES docs](https://www.gnu.org/software/automake/manual/html_node/Sources.html#Sources) and the linked example, and I'm not convinced that this would help us. I *do* want to keep the dependencies of "<build js thing>: package-lock.json", so that any time the node_modules change, things get rebuilt correctly. So BUILT_SOURCES would just be one extra declaration that doesn't replace anything; it would only help for the "build from scratch" case to push the npm install to the front, but we only need either that or calling it in autogen.sh (and you seem to prefer the latter?)